### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by rafahop**
Initial boilerplate generation, created via dp project generation tool

### CODE REVIEW

**Review by bpathak-ons — COMMENTED** [review]
lint target, as a placeholder, needs to be defined

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
-# dp-in-memory-mongodb
+dp-in-memory-mongodb
+================
```
Comment (gedge) [inline README.md:2]: this 'underline' format is not our preferred style (use `# dp-in-memory-mongodb` for the previous line, instead)
also, it was inconsistent with the rest of the file (which uses the `# header` style)

Comment (gedge) [reply README.md:2]: I strongly suggest a markdown-linter

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
-# dp-in-memory-mongodb
+dp-in-memory-mongodb
```
Comment (gedge) [inline README.md:1]: I would have preferred `dp-mongodb-in-memory` as the repo name

- follows big-endian naming scheme
- would list alongside other `dp-mongodb*` repos

too late to rename this repo?? 🙏

**Comment by rafahop** [comment]
I am changing the project generator tool to use the right readme format and include the lint job. I will delete this repo and create it again using the new version with the suggested new name

**Pull request closed without merging**